simple_python_with_bugs.py

```diff
@@ -2,6 +2,9 @@ def atoi(s: str) -> int:
     if not s:
         raise ValueError("Input string is empty")
 
+    if "." in s:
+        raise RuntimeError("Input string is a float, only integers are supported")
```

Comment:
RuntimeError is not the right error here. 
Please take a look at the other errors raised in this function and also raise ValueError in case of an unexpected input